great_expectations/rule_based_profiler/data_assistant/volume_data_assistant.py

```diff
@@ -46,8 +35,11 @@ def __init__(
         batch_request: Union[BatchRequestBase, dict],
         data_context: BaseDataContext = None,
     ):
+        data_assistant_result_cls = VolumeDataAssistantResult
```

Comment:
@NathanFarmer same comment as earlier...

Reply:
✅ Taken care of offline